src/command_line_parser.h

```diff
@@ -57,7 +59,6 @@ struct PerfAnalyzerParameters {
   std::unordered_map<std::string, std::vector<int64_t>> input_shapes;
   std::vector<cb::ModelIdentifier> bls_composing_models;
   uint64_t measurement_window_ms = 5000;
-  bool using_concurrency_range = false;
```

Comment:
This was previously used for the deprecated options.

Deprecated options now leverage existing options' state.